File: CPP/Factors.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, i;
    cout<<"Enter the number: ";
    cin>>n;

    cout<<"The factors of "<<n<<" are:"<<endl;      //endl for \n
    for(i=1; i<=n; i++)
    {
        if(n%i==0)
        {
            printf("%d ", i);
        }
    }
    cout<<endl;
    return 0;
}

File: Java/Factors.java

import java.util.*;

public class Factors {
    public static void main(String[] args) {
        int n, i;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number: ");
        n = sc.nextInt();

        System.out.println("The factors of " + n + " are:");
        for (i = 1; i <= n; i++) {
            if (n % i == 0) {
                System.out.print(i + " ");
            }
        }
        System.out.println();
        sc.close();
    }
}
